src/components/display/pluginLink.tsx

```diff
@@ -0,0 +1,33 @@
+import { PluginUiMessageHandler } from '@cloudquery/plugin-config-ui-connector';
+import Link, { LinkProps } from '@mui/material/Link';
+import React, { ReactNode } from 'react';
+
+/**
+ * @public
+ */
+export type PluginLinkProps = Omit<LinkProps, 'onClick' | 'children' | 'href'> & {
+  children: ReactNode;
+  href: string;
+  pluginUiMessageHandler: PluginUiMessageHandler;
+}
+
+/**
+ * Wrapper for MUI Link component that opens a URL in the plugin UI top app.
+ *
+ * @public
+ */
+export function PluginLink({ children, href, pluginUiMessageHandler, ...linkProps }: PluginLinkProps) {
```

Comment:
Can we name it just `Link`? When I see `PluginLink` I immediately think that this is somehow related to the plugin itself, maybe it's a link to plugin page or something. I think the library should expose as clear outputs as possible while the code importing it should decide on its own if it wants to use an alias for it

Reply:
thought it might be annoying to differentiate on import, but I see the value 👍

Reply:
yeah, sounds like I need to create another eslint rule)